### ISSUE TITLE
1.12 Random massive server log spam

### ISSUE THREAD

**Issue by Thedrummonger** [Thutmose/Pokecube#205]
When playing on my personal server using a custom modpack with this mod I randomly get massive server log spam that freezes the game for about 10 seconds until I'm kicked for server timeout. After I'm kicked the spam stops and i'm free to rejoin. Here is a snippet of the log (basically everything that was still in the command line window when it stopped. None of my server log files seem to have this information in it.) https://pastebin.com/Dpb5KPj3 
The problem does not seem to persist if I remove Pokecube alternative, revival, core and mobs. I'm not super great at deciphering log files but at first glance it seems to say something about Pokemon spawning so I feel I should note my config is set to only allow spawning of pokemobs in dimension 111 (Lost cities). This issue is only happening in the overworld and I have not visited dimension 111 yet in this world. Not sure if that has anything to do with it.
Thanks!

Pokecube Core: 6.14.1
Pokecube Revival (if applicable): 6.8.2
Pokecube Mobs (if applicable): 2.14.0
Minecraft: 1.12
Forge: 14.21.1.2426

**Comment by Thutmose** [comment]
Hmm, I will check if something has gone screwy with the blacklist/whitelist then, it sounds like something has, I will try to see if I can find anything in that spam to indicate the issue.

**Comment by Thutmose** [comment]
I think it might be something wrong with the trainers? can you try setting trainerSpawn to false in pokecube_adventures.cfg and see if it still occurs? if not, then at least I know where to look.

**Comment by Thutmose** [comment]
in my testing with lost cities, whitelist and stuff, there don't seem to be any issues, I will continue to look for it though.

**Comment by Thedrummonger** [comment]
I disabled trainer spawn and it still seems to be happening unfortunately. Here is another log snipet if it helps any https://pastebin.com/SCC7YNeg. For the most part it just seems to be repeating the same thing over and over again with only this line changing
[17:28:14] [Server thread/INFO] [STDERR]: [pokecube.core.PokecubeCore:createPoke
mob:276]: class pokecube.core.entity.pokemobs.GenericPokemob270 Lotad
Every time this line shows in the console it's a different pokemon name (Lotad in this case). It almost seems like it's looping through every pokemon in the game.

I also tried deleting the config file and booting the server with default config. Doing this for whatever reason made the log spam still happen, but happen a lot slower. Slow enough that I could Isolate the error that was being repeated. Here's that https://pastebin.com/yFAWDaUM. The strangest part about this is since the configs were untouched pokemon should be to spawn in the overworld but I was unable to find any spawning. It seemed that whenever one tried to spawn, they failed to spawn and gave that error in the server console. 

Since this seems to be completely blocking spawns, it seems to me like this might not be your fault but the fault of another mod due to that fact that I seem to be the only one having this issue :P. I'll keep looking into it and let you know any information I find. Thanks for your time!
